### PR TITLE
Jenkins pipelines now use persistent local maven repo.

### PR DESCRIPTION
**Pull request by iocanel**

### CODE REVIEW

**Comment by iocanel** [comment]
retest this please

**Comment by jimmidyson** [comment]
@iocanel Should we merge or close this PR?

**Comment by KurtStam** [comment]
retest this please

File: Jenkinsfile [hunk 1]
```diff
@@ -19,6 +17,7 @@ slave {
                     test(component: 'ipaas-rest', namespace: "${testingNamespace}", serviceAccount: 'jenkins')
                  }
 
+            }
         }
     }
 }
```
Comment (zregvart) [inline Jenkinsfile]: This `}` is superfluous

**Comment by zregvart** [comment]
I think the build in the internal Jenkins [does not succeed](https://jenkins-syndesis-ci.b6ff.rh-idev.openshiftapps.com/job/syndesis-rest/view/change-requests/job/PR-365/3/console) because `GITHUB_OAUTH_CLIENT_ID` and `GITHUB_OAUTH_CLIENT_SECRET` are not defined but are required for system tests, I think those can be added by the following pipeline magic:

    def GITHUB_OAUTH_CLIENT_ID
    def GITHUB_OAUTH_CLIENT_SECRET
    
    node('master') {
        GITHUB_OAUTH_CLIENT_ID = env.GITHUB_OAUTH_CLIENT_ID
        GITHUB_OAUTH_CLIENT_SECRET = env.GITHUB_OAUTH_CLIENT_SECRET
    }
    
    slave {
        withOpenshift {
            withMaven(envVars: [
                    containerEnvVar(key: 'GITHUB_OAUTH_CLIENT_ID', value: GITHUB_OAUTH_CLIENT_ID),
                    containerEnvVar(key: 'GITHUB_OAUTH_CLIENT_SECRET', value: GITHUB_OAUTH_CLIENT_SECRET)
                ]) {
                inside {
    ...
    }

This steals those variables from Jenkins master and passes them to the container spun up for the build. Perhaps these should be in the pipeline library? @iocanel what do you think?

**Comment by zregvart** [comment]
retest this please

**Comment by KurtStam** [comment]
retest this please

**Comment by KurtStam** [comment]
@iocanel what is the status of this?

**Comment by iocanel** [comment]
I am closing this one, and will create a new one when things settle down.

**Comment by iocanel** [comment]
I'll create a new one

**Pull request closed without merging**